slsim/Sources/SupernovaeCatalog/supernovae_sample.py

```diff
@@ -161,21 +166,23 @@ def supernovae_catalog(self, host_galaxy=True, lightcurve=True):
         """
         sne_lightcone = SNeLightcone(
             self.cosmo,
-            redshifts=np.linspace(0, 2.379, 100),
+            redshifts=np.linspace(0, 5, 500),
```

Comment:
you hard code z=5 as the max redshift of the SNe. Perhaps you can do this as a default input to this class? Makes it more transparent what happens.